crates/ruff_linter/src/rules/pycodestyle/rules/invalid_escape_sequence.rs

```diff
@@ -70,39 +72,16 @@ pub(crate) fn invalid_escape_sequence(checker: &Checker, string_like: StringLike
             StringLikePart::String(_) | StringLikePart::Bytes(_) => {
                 analyze_escape_chars(locator, part.range(), part.flags())
             }
-            StringLikePart::FString(f_string) => {
-                let flags = AnyStringFlags::from(f_string.flags);
-                let mut escape_chars_state = EscapeCharsState::default();
-                // Whether we suggest converting to a raw string or
-                // adding backslashes depends on the presence of valid
-                // escape characters in the entire f-string. Therefore,
-                // we must analyze escape characters in each f-string
-                // element before pushing a diagnostic and fix.
-                for element in &f_string.elements {
-                    match element {
-                        FStringElement::Literal(literal) => {
-                            escape_chars_state.update(analyze_escape_chars(
-                                locator,
-                                literal.range(),
-                                flags,
-                            ));
-                        }
-                        FStringElement::Expression(expression) => {
-                            let Some(format_spec) = expression.format_spec.as_ref() else {
-                                continue;
-                            };
-                            for literal in format_spec.elements.literals() {
-                                escape_chars_state.update(analyze_escape_chars(
-                                    locator,
-                                    literal.range(),
-                                    flags,
-                                ));
-                            }
-                        }
-                    }
-                }
-                escape_chars_state
-            }
+            StringLikePart::FString(f_string) => analyze_escape_chars_in_interpolated_string(
```

Comment:
I'm a bit concerned about the wild mix of terminology used in this PR. I'd prefer if we sticked to one, even if it is `FT` but we should avoid using different terms for the same thing.